packages/fast-components-styles-msft/src/select-option/index.ts

```diff
@@ -0,0 +1,122 @@
+    contrast,
+    Direction,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import { curry } from "lodash-es";
+
+const styles: ComponentStyles<SelectOptionClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<SelectOptionClassNameContract, DesignSystem> => {
+    type ContrastFunction = (operandColor: string, referenceColor: string) => string;
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+    const contrastScale: number = designSystem.contrast;
+    const brandColor: string = designSystem.brandColor;
+    const color: string = designSystem.foregroundColor;
+    const scaledEnsureNormalContrast: ContrastFunction = curry(ensureNormalContrast)(
+        contrastScale
+    );
+    const primaryRestBackgroundColor: string = scaledEnsureNormalContrast(
+        scaledEnsureNormalContrast(brandColor, designSystem.backgroundColor),
+        color
+    );
+    const primaryDisabledBackground: string = disabledContrast(
+        contrastScale,
+        primaryRestBackgroundColor,
+        designSystem.backgroundColor
+    );
+    const primaryDisabledColor: string = disabledContrast(
+        contrastScale,
+        color,
+        primaryDisabledBackground
+    );
+    const primarySelectedBackground: string = contrast(
+        1.7,
+        designSystem.foregroundColor,
+        designSystem.backgroundColor
+    );
+    return {
+        selectOption: {
+            listStyleType: "none",
+            height: density(32),
+            display: "grid",
+            gridTemplateColumns: `${applyLocalizedProperty(
+                "12px auto auto 1fr 12px",
+                "12px 1fr auto auto 12px",
+                direction
+            )}`,
+            gridTemplateRows: "auto",
+            alignItems: "center",
+            padding: "0",
+            margin: "0 4px",
+            ...typographyPattern.rest,
+            whiteSpace: "nowrap",
+            overflow: "hidden",
+            cursor: "default",
+            ...applyTypeRampConfig("t7"),
+            background: designSystem.backgroundColor,
+            borderRadius: toPx(designSystem.cornerRadius),
+            border: "2px solid transparent",
+            ...applyFocusVisible({
+                borderColor: ensureForegroundNormal,
+            }),
+            "&:hover": {
+                background: hoverContrast(
+                    designSystem.contrast,
+                    designSystem.backgroundColor
+                ),
+            },
+        },
+        selectOption_contentRegion: {
+            gridColumnStart: "3",
+            overflow: "hidden",
+            textOverflow: "ellipsis",
+        },
+        selectOption_glyph: {
+            gridColumnStart: `${applyLocalizedProperty("2", "4", direction)}`,
+            display: "inline-block",
+            position: "relative",
+            maxWidth: "16px",
+            margin: "0 12px 0 0",
```

Comment:
per https://developer.mozilla.org/en-US/docs/Web/CSS/margin this is adding a margin-right value, please add RTL

Reply:
yep